src/content/tools/devtools/app-size.md

```diff
@@ -267,7 +267,7 @@ the Wonderous App using DevTools, check out the
 [App Size Tool tutorial][app-size-tutorial]. Various strategies
 to reduce an app's size are also discussed.
 
-[Use the treemap]: #using-the-treemap
+[Use the treemap]: #use-the-treemap
```

Comment:
Won't this change break incoming references?

Reply:
If there are any, yup.